build.gradle

```diff
@@ -99,8 +99,8 @@ subprojects { subproject ->
 		log4j2Version = '2.7'
 		logbackVersion = '1.1.7'
 		mockitoVersion = '1.10.19'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.5'
-		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '4.0.0'
```

Comment:
The AMQP client version's defined in the Platform. Brussels (based on Boot 1.5) has 3.6.5 in it at the moment. We could jump to 4.0 if necessary, but it does strike me as a little odd for AMQP 1.7 to require a new major version of the AMQP client. You'd know much better than me if that's a reasonable upgrade to be making, though.